pkg/model/sdk_helper.go

```diff
@@ -382,20 +386,26 @@ func (a *SDKAPI) SDKAPIInterfaceTypeName() string {
 }
 
 // Override the operation type and/or resource name if specified in config
-func getOpTypeAndResourceName(opID string, cfg *ackgenconfig.Config) (OpType, string) {
+func getOpTypeAndResourceName(opID string, cfg *ackgenconfig.Config) ([]OpType, string) {
+	var unmarshaledOpTypes ackgenconfig.StringArray
 	opType, resName := GetOpTypeAndResourceNameFromOpID(opID)
+	opTypes := []OpType{opType}
 
-	if cfg != nil {
-		if operationConfig, exists := cfg.Operations[opID]; exists {
-			if operationConfig.OperationType != "" {
-				opType = OpTypeFromString(operationConfig.OperationType)
-			}
+	if cfg == nil {
+		return opTypes, resName
+	}
+	if operationConfig, exists := cfg.Operations[opID]; exists {
+		if operationConfig.ResourceName != "" {
+			resName = operationConfig.ResourceName
+		}
 
-			if operationConfig.ResourceName != "" {
-				resName = operationConfig.ResourceName
+		if b, err := json.Marshal(operationConfig.OperationType); err == nil {
```

Comment:
Thinking a second time about this, i think that you don't have to call `json.Marshal`/`json.Unmarshall` here - because `StringArray.UnmarshalJSON` will be called when `model.Model` is getting initialized https://github.com/aws-controllers-k8s/code-generator/blob/main/pkg/model/model.go#L700

Reply:
Yep you are right :)

Reply:
In this case, is operation.go the right place for the UnmarshalJSON method ?

Reply:
Yes it makes sense to have the type and it methods on the same file so it's good!